src/Chapter1.hs

```diff
@@ -640,8 +646,10 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+  | div n 10 == 0 = n
```

Comment:
`div` on negative numbers returns negative number:

```haskell
ghci> div (-1) 10
-1
```

That's why this function hangs on CI and runs into an infinite loop. So, to make tests pass, you need to handle corner cases in this function :slightly_smiling_face: